File: Python/program80.py
# Input : 4
# Output: 4 * 3 * 2 *  1 * 

def Display(iNo):
    if(iNo < 0):
        iNo = -iNo
        
    for i in range(iNo , 0, -1 ):
        print(i,"\t*\t", end="")
    print()
    
def main():
    print("Enter number")
    iValue = int (input())
    
    Display(iValue)
    
    
if __name__ == "__main__":
    main()

File: C++/program80.cpp
//Iteration to Iteration

#include<iostream>
using namespace std;

void Display()
{
    int j = 0, i = 0;
    
    for(i = 0; i < 3; i++)
    {
        for(j =1; j <= 3; j++)
        {
            cout<<"*"<<'\t'; 
        }
        cout<<'\n';
    }
   



}

int main()
{

    Display();

    return 0;
}